workflows/inngest/src/workflow-factory.test.ts

```
@@ -0,0 +1,446 @@
+    getInitData: false,
+    errorCauseChain: false,
+    // Storage round-trip test - enabled since storage tests pass
+    errorStorageRoundtrip: false,
+    // Error persistence tests - enabled with storage spy access
+    errorPersistWithoutStack: false,
+    errorPersistMastraError: false,
+    // Resume tests - enabled for testing
+    resumeBasic: false,
+    resumeWithLabel: false, // Testing - uses label instead of step
+    resumeWithState: true, // requestContext bug #4442 - request context not preserved during resume
+    resumeNested: true, // Not yet implemented
```

Biome (2.3.14)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)